tests/Fixtures/ConvertConfig/features/one.feature

```diff
@@ -0,0 +1 @@
+Feature: test
```

Comment:
When testing the `paths` settings, these need to point to existing files, so I added a couple of test feature files to be able to test it